src/core/census/rpc_metric_id.h

```diff
@@ -31,16 +31,21 @@
  *
  */
 
-#ifndef CENSUS_RPC_STAT_ID_H
-#define CENSUS_RPC_STAT_ID_H
+#ifndef CENSUS_RPC_METRIC_ID_H
+#define CENSUS_RPC_METRIC_ID_H
 
-/* Stats ID's used for RPC measurements. */
-#define CENSUS_INVALID_STAT_ID 0     /* ID 0 is always invalid */
-#define CENSUS_RPC_CLIENT_REQUESTS 1 /* Count of client requests sent. */
-#define CENSUS_RPC_SERVER_REQUESTS 2 /* Count of server requests sent. */
-#define CENSUS_RPC_CLIENT_ERRORS 3   /* Client error counts. */
-#define CENSUS_RPC_SERVER_ERRORS 4   /* Server error counts. */
-#define CENSUS_RPC_CLIENT_LATENCY 5  /* Client side request latency. */
-#define CENSUS_RPC_SERVER_LATENCY 6  /* Server side request latency. */
+/* Metric ID's used for RPC measurements. */
+/* Count of client requests sent. */
```

Comment:
Just like the default/stale handle, one use case is to answer the query "how many record_stats() calls have invalid metric id across different processes?", we can use it as a measure of the quality of stats.

There are various scenarios that two processes (P_A and P_B) may see different metric id definitions (e.g. They are running different versions of binaries, or, there's a long latency to get updated metric ids from a centralized server). Having an common INVALID_METRIC_ID can allow us to globally collect and aggregate the stats of "record against invalid metric id".